### PR TITLE
Added descriptions to the example

### PR DESCRIPTION
**Pull request by DavidMansolino**
Added descriptions to the example.

### CODE REVIEW

**Comment by fabienrohrer** [comment]
I know that I'm fussy, but I'd prefer a lot the using the C++ streams/strings to display this.

Otherwise these comments seems to be good to me.

**Comment by DavidMansolino** [comment]
No problem.

File: projects/robots/darwin-op/controllers/sample/Sample.cpp [hunk 1]
```diff
@@ -104,8 +104,13 @@ bool Sample::getBallCenter(double &x, double &y) {
 
 // function containing the main feedback loop
 void Sample::run() {
-    // First step to update sensors values
-    myStep();
+	
+  cout << printf("---------------Demo of DARwIn-OP---------------" << endl;
```
Comment (fabienrohrer) [inline projects/robots/darwin-op/controllers/sample/Sample.cpp]: cout << printf
;-)

Comment (DavidMansolino) [inline projects/robots/darwin-op/controllers/sample/Sample.cpp]: oups ...

**Pull request merged**